packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -30,7 +30,9 @@ export function getSPOAndGraphRequestIdsFromResponse(headers: { get: (id: string
     interface LoggingHeader {
         headerName: string;
         logName: string;
+        mapping?: Map<string, string | number | boolean>;
```

Comment:
I'm not sure what this argument is about - it's not used, and nobody provides it

Reply:
I use it to map the x-fluid-telemetry field below (mapping). I was initially adding a new interface specifically for the x-fluid-telemetry but it can be eventually used on the headersToLog as well. So I simply added the mapping property as optional.

Reply:
Ah, I see it (missed it). I'd say - I do not see much value in adding it here.
You can simply remove this addition and do 

const fluidKeyValuesToLog = {...} 

below without any type
I.e. I'm not even sure why this type is related to your changes, it does not affect how actual first-level properties are passed.